### DISCUSSION TITLE
有没有v4版的纯英文检测模型？

### DISCUSSION THREAD

**Discussion by nissansz** [PaddlePaddle/PaddleOCR#14240] [Q&A]
有没有v4版的纯英文检测模型？

**Comment by GreatV** [comment]
没有

**Reply by nissansz** [reply]
按英文单词检测效果最好的模型是用哪个？

**Reply by GreatV** [reply]
不能按单词检测

**Comment by nissansz** [comment]
识别前检测出的文本框，是单词级别短句的准确率高还是长句？









--------------------------------------------------------------------------------





------------------ 原始邮件 ------------------




 
 发件人: Wang Xin  ***@***.***>
 

 
 发送时间: 2024-11-17 11:00:07
 

 
 收件人:PaddlePaddle/PaddleOCR ***@***.***> 

 
 抄送:nissanjp ***@***.***>,Author ***@***.***> 

 
 主题: Re: [PaddlePaddle/PaddleOCR] 有没有v4版的纯英文检测模型？ (Discussion #14240)
 





 不能按单词检测

**Reply by nissansz** [reply]
那个模型检测效果接近下面的？

![00402f1b99e51034cf7e8699e6e5660](https://github.com/user-attachments/assets/f5bf102e-0646-45a8-9ae7-f38b6101c4ce)
